Makefile.in

```diff
@@ -310,8 +310,11 @@ test_high_level_test_deps = test_fishscript test_interactive
 active_test_goals = $(filter $(test_goals),$(foreach a,$(or $(MAKECMDGOALS),$(.DEFAULT_GOAL)),$(a) $($(a)_test_deps)))
 filter_up_to = $(eval b:=1)$(foreach a,$(2),$(and $(b),$(if $(subst $(1),,$(a)),$(a),$(eval b:=))))
 
+# The values for XDG_DATA_HOME and XDG_CONFIG_HOME mimic those from tests/test_util.fish.
 test_low_level: fish_tests $(call filter_up_to,test_low_level,$(active_test_goals))
-	./fish_tests
+	rm -rf test/data test/home
```

Comment:
> Probably remove the test data when tests are done.

That's a bad idea. If tests fail it is useful to have the transient test data available to debug the failure. On the other hand you might reasonably think the `rm` command I added is superfluous since `make test-prep` should remove those files. But that doesn't happen if someone runs `make test_low_level` which skips the `make test-prep` target. It's not obvious how best to deal with the conflicting expectations of the various make test targets.